src/request/components/request-detail-panel-logging.tsx

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+import { ILogMessage } from '../messages/ILogMessage';
+import { ILoggingViewModel } from '../view-models/ILoggingViewModel';
+
+import React = require('react');
+
+interface ILogMessagesProps {
+    messages: ILogMessage[];
+}
+
+/**
+ * React class to display console messages
+ */
+class LogMessages extends React.Component<ILogMessagesProps, {}> {
```

Comment:
In this case because there is no reuse else where and the complexity is low, it would probably be simpler to read if it was just one component but its not a big deal.